Sources/TokamakCore/Fiber/ViewGeometry.swift

```diff
@@ -0,0 +1,55 @@
+// Copyright 2022 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Carson Katri on 2/17/22.
+//
+
+import Foundation
+
+public struct ViewGeometry: Equatable {
+  @_spi(TokamakCore) public let origin: ViewOrigin
+  @_spi(TokamakCore) public let dimensions: ViewDimensions
```

Comment:
nit: formatting tweaks for easier skimming through these declarations
```suggestion
  @_spi(TokamakCore) 
  public let origin: ViewOrigin

  @_spi(TokamakCore) 
  public let dimensions: ViewDimensions
```